static/js/components/Loading.js

```diff
@@ -14,6 +18,8 @@ type SpinnerProps = {
   className?: string
 }
 
+const emptyPostsToRender = 5
```

Comment:
maybe this should go in the new `lib/constants.js` file too?

Reply:
It's only relevant to the post loading component in this file, so I thought it was better to put it here. I put it in a constant instead of inlining it just to make it more obvious where the count was coming from, and easier to find and change